.github/workflows/ci.yml

```diff
@@ -33,7 +33,41 @@ jobs:
 
   integration-test:
     needs: poetry-install
-    uses: ./.github/workflows/integration-test.yml
+    # Exit if its a commit from Gitbook
+    if: ${{ ! startsWith(github.event.head_commit.message, 'GitBook:') }}
```

Comment:
I took this from the previous integration testing github workflow. I don't want to touch this unless I absolutely have to, because it's not related to my changes and I'm don't 100% understand how it works.